sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ComponentDescriptorFactory.scala

```diff
@@ -179,6 +193,28 @@ private[impl] object ComponentDescriptorFactory {
     Eventing.newBuilder().setIn(eventSource).build()
   }
 
+  def eventingInForEventSourcedEntityServiceLevel(clazz: Class[_]): Option[kalix.ServiceOptions] = {
+    eventSourcedEntitySubscription(clazz).map { ann =>
+
+      val entityType = findEventSourcedEntityType(clazz)
+
+      val in = EventSource
+        .newBuilder()
+        .setIgnoreUnknown(ann.ignoreUnknown())
```

Comment:
I remember not adding `ignoredUnknown` on the descriptor when working with subscriptions. I had to do this because otherwise, the proxy ignores all messages for the Spring SDK. I tried to find the source code of this in the proxy but couldn't find it. Could you keep me honest here @johanandren?

Reply:
Good catch! You are right. 

The ignore should be propagated to the proxy, but kept and handled on the SDK side. I will change it.